ams/ResetCommand.cs

```diff
@@ -32,7 +32,16 @@ internal class ResetCommand : BaseMigrator
 
         public override async Task MigrateAsync(CancellationToken cancellationToken)
         {
-            var account = await GetMediaAccountAsync(_options.AccountName, cancellationToken);
+            MediaServicesAccountResource? account = null;
+            try
+            {
+                account = await GetMediaAccountAsync(_options.AccountName, cancellationToken);
+            }
+            catch (Exception)
+            {
+                _logger.LogError("No valid media account was found.");
+                return;
```

Comment:
Do not swallow exceptions.  Throw them.  Why should you continue if account is not found.